sdk/core/Azure.Core.TestFramework/tests/ClientTestBaseDiagnosticScopeTests.cs

```diff
@@ -117,7 +117,7 @@ public class InvalidDiagnosticScopeTestClient
             private DiagnosticScope CreateScope(string method)
             {
                 // intentionally does not suppress nested activities
-                DiagnosticScopeFactory clientDiagnostics = new DiagnosticScopeFactory("Azure.Core.Tests", "random", true, suppressNestedClientActivities: false);
```

Comment:
nit: why remove the named parameter?

Reply:
I was not going to provide a default value for `isStable` flag and didn't want to change the order of parameters. But decided to actually provide a default `false` for `isStable`.